iphone/Maps/ar.lproj/Localizable.strings

```diff
@@ -1723,3 +1723,39 @@
 "traffic_outdated" = "Traffic data is outdated.";
 
 "big_font" = "Bigger labels";
+
+"sponsored_delivery_club_ru_header" = "Order food delivery";
+
+"sponsored_lamoda_ua_header" = "Online clothing and shoe store";
+
+"sponsored_lamoda_ru_header" = "Online clothing and shoe store";
+
+"sponsored_tutu_ru_header" = "Train tickets online.";
+
+"sponsored_geerbest_uk_header" = "Online electronics store";
+
+"sponsored_geerbest_fr_header" = "Online electronics store";
+
+"sponsored_geerbest_ge_header" = "Online electronics store";
+
+"sponsored_rentalcars_ww_header" = "Booking cars online";
+
+"sponsored_viator_ww_header" = "Online trip search";
+
+"sponsored_delivery_club_ru_description" = "Fast delivery from 4,000 restaurants. Order from Delivery Club.";
+
+"sponsored_lamoda_ua_description" = "More than 1,000 brands. Up to 70% off. Free delivery to all parts of Ukraine.";
+
+"sponsored_lamoda_ru_description" = "More than 1,000 brands. Up to 70% off. Free delivery to all parts of Russia.";
```

Comment:
"sponsored_lamoda_description" = "More than 1,000 brands. Up to 70% off. Free delivery to all parts of %s."